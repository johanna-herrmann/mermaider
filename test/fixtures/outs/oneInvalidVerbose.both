invalid.md Error
invalid.md: Syntax error in graph
simple.md OK
