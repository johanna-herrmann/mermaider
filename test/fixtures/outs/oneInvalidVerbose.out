invalid.md Error
simple.md OK
